maintenance/wikia/migrateImagesBetweenSwiftDC.php

```diff
@@ -193,6 +187,8 @@ private function store() {
 
 		if ( !$srcStorage->exists( $remoteFile ) ) {
 			$this->output( "\tCannot find image to sync \n" );
+			$this->imageSyncQueue->setError( self::ERROR_CANT_FIND_FILE );
```

Comment:
maybe it could be renamed to `imageSyncQueueItem` using automatic refactoring? if not forget about it